12/umbraco-cms/fundamentals/backoffice/settings-dashboards.md

```diff
@@ -30,7 +30,7 @@ For more information about Examine Management, see the [Examine Management](../.
 
 <summary>Published Status</summary>
 
-The Published Status dashboard displays the status of your site in the Published Cache Status section alongwith the Content and Media nodes value. The Caches section provides three options: Memory Cache, Database Cache, and Internals (NuCache).
+The Published Status dashboard displays the status of your site in the Published Cache Status section alongside the Content and Media nodes value. The Caches section provides three options: Memory Cache, Database Cache, and Internals (NuCache).
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?